### DISCUSSION TITLE
DPF generation

### DISCUSSION THREAD

**Discussion by djeddy** [agronick/aa-torque#38] [Ideas]
Is it possible to have the PID that has the dpf generation percentage?  Can you tell me where to get other themes?  Thank you

**Comment by agronick** [comment]
The PID will depend on your car. If it is a Mazda there is a plugin for that. The only themes are the ones that are included.

**Reply by djeddy** [reply]
the car in question is an Alfa Romeo Stelvio

**Comment by cffilippo** [comment]
hi what would it be for mazda?
